conductor/lib/uploader_v2.py

```diff
@@ -299,7 +317,7 @@ def run(self):
         # call stop method to properly shutdown child processes, cleanup, etc
         self.stop()
 
-        LOGGER.debug("Exiting process")
+        self.log("info", "eXITING_PROCESS")
```

Comment:
`"eXISTING"` vs `"EXISTING"`.  Not sure if this is intentional, as it looks like there is a similar implementation in line 444: `self.log("info", "sTARTING_UPLOAD")`